main.go

```diff
@@ -60,15 +74,16 @@ func main() {
 	}
 
 	// TODO: Accept other logger inputs
-	log.SetHandler(text.New(os.Stderr))
-
-	log.Infof("Starting git2consul version: %s", version.Version)
-
-	if len(filename) == 0 {
-		log.Error("No configuration file provided")
-		os.Exit(ExitCodeFlagError)
+	switch logfmt {
+	case "text":
+		log.SetHandler(text.New(os.Stderr))
+	case "json":
+		log.SetHandler(json.New(os.Stderr))
 	}
 
+	//log.Infof("Starting git2consul version: %s", version.Version)
```

Comment:
Please remove this line of commented code.

Reply:
thank you @seanmalloy  removed the line